.coveragerc

```diff
@@ -1,8 +1,14 @@
 [run]
 branch = True
 source = dallinger
+parallel = True
 
 [report]
-fail_under = 41
+fail_under = 55
 show_missing = True
 skip_covered = True
+
+[paths]
+source =
+    dallinger
+    .tox/py27/lib/python2.7/site-packages/dallinger
```

Comment:
This allows coverage reports for tests in the .tox subdirectory to be combined with matching tests from `dallinger` when combine is specified in tox.ini so the same tests aren't reported twice?